uploadfs.js

```diff
@@ -65,16 +64,45 @@ function Uploadfs() {
       self.cdn = options.cdn;
     }
 
+
     // Load image backend
-    self._image = options.image;
+     self._image = options.image;
+    const processors = [ 'imagemagick', 'jimp' ];
     if (typeof (self._image) === 'string') {
-      self._image = require('./lib/image/' + self._image + '.js')();
+      if (processors.includes(self._image)) {
+        console.error('We now recommend the use of the sharp.js library for image processing. As fallback, if sharp is not installed we will attempt to use imagemagick instead.');
+      } else if (self._image != 'sharp') {
+        return callback('The sharp.js library is the default with imagemagick as the fallback. No other image processors are supported');
```

Comment:
This should still be a warning and not a failure, with the same fallback, since we're pulling a bit of a fast one as it is by removing some options.

Reply:
We should only force sharp if they (1) specify no image backend, or (2) specify one we completely removed (jimp or imagecrunch). If they want to explicitly specify imagemagick then they know what they are doing.

Reply:
All new logic section addresses this.